api/capi/include/nntrainer.h

```diff
@@ -215,6 +215,43 @@ int ml_train_model_set_optimizer(ml_train_model_h model,
 int ml_train_model_set_dataset(ml_train_model_h model,
                                ml_train_dataset_h dataset);
 
+/**
+ * @brief Save the model after training
+ * @details Use this function to save the current model
+ *
+ * @since_tizen 6.5
+ * @param[in] model The NNTrainer model handle to save
+ * @param[in] path_prefix  Path prefix to save the file. This function will save
+ * one or number of files using the given prefix. Note that this does not state
+ * the exact file name but prefix
+ * @param[in] option Option flag which part of model should be saved
+ * @return @c 0 on success, Otherwise a negative error value
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER Path is a directory without a prefix, or
+ * the given path is invalid.
+ * @retval #ML_ERROR_NOT_SUPPORTED Given flag is not valid for the model
+ */
+int ml_train_model_save(ml_train_model_h model, const char *path_prefix,
```

Comment:
please add details for which state model should be in to support this - constructed, compiled, at least run once, etc.

Reply:
Done